daemon/cmd/daemon_main.go

```diff
@@ -1822,28 +1825,10 @@ func runDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *daem
 		}
 	}
 
-	// Process queued deletion requests
-	bootstrapStats.deleteQueue.Start()
```

Comment:
Where did the bootstrapStats go? I'm not finding them.

Reply:
Oops, forgot to move it to `deletion_queue.go` start hook. Fixed.